indexer-app/src/cmr/indexer/api/routes.clj

```diff
@@ -35,34 +31,34 @@
 (def ^:private index-set-routes
   "Routes providing index-set operations"
   (context "/index-sets" []
-    (POST "/" {body :body request-context :request-context params :params headers :headers}
```

Comment:
why did we get rid of the params and headers for this endpoint?

Reply:
kondo told me to